Automattic/wp-calypso
# Nav Unification: Add "Manage Blogs" menu in the "Me" section

#### Changes proposed in this Pull Request

Adds a new "Manage Blogs" menu in the "Me" section that links to the "My Blogs" page in where users can manage the blogs they have access, including features as "Post by Email" and "Post by Voice".

<img width="289" alt="Screen Shot 2021-02-26 at 13 56 50" src="https://user-images.githubusercontent.com/1233880/109310505-ea5ced80-7844-11eb-9c8a-0baaf3923adf.png">

Since "My Blogs" lives on a different site (`dashboard.wordpress.com`), the link is opened in a new tab so users don't lose their current navigation.

#### Testing instructions

_You may prefer to use Firefox in order to avoid an annoying bug in Chrome regarding the 2FA codes._
- Using the Calypso live link below, go to `/me`.
- Make sure there is a "Manage Blogs" menu that links to "My Blogs".
- Disable the Nav Unification by appending the `?disable-nav-unification` query param to the URL.
- Make sure the "Manage Blogs" menu is not added.

Fixes https://github.com/Automattic/wp-calypso/issues/49453
